HACKING.rst.txt

```diff
@@ -194,289 +219,6 @@ project and SciPy the community, and it can always use a new pair of hands.
-Some IDEs (`Spyder`_ for example) have utilities to manage PYTHONPATH.  On Linux
-and OSX, you can run the command::
-
-    $ export PYTHONPATH=$PWD
-
-and on Windows
-
-    $ set PYTHONPATH=/path/to/scipy
-
-Now editing a Python source file in SciPy allows you to immediately
-test and use your changes (in ``.py`` files), by simply restarting the
-interpreter.
-
-
-*Are there any video examples for installing from source, setting up a
-development environment, etc...?*
-
-Currently, there are two video demonstrations for Anaconda Python on macOS:
-
-`Anaconda SciPy Dev Part I (macOS)`_ is a four-minute
-overview of installing Anaconda, building SciPy from source, and testing
-changes made to SciPy from the `Spyder`_ IDE.
-
-`Anaconda SciPy Dev Part II (macOS)`_ shows how to use
-a virtual environment to easily switch between the "pre-built version" of SciPy
-installed with Anaconda and your "source-built version" of SciPy created
-according to Part I.
-
-
-*Are there any video examples of the basic development workflow?*
-
-`SciPy Development Workflow`_ is a five-minute example of fixing a bug and
-submitting a pull request. While it's intended as a followup to
-`Anaconda SciPy Dev Part I (macOS)`_ and `Anaconda SciPy Dev Part II (macOS)`_,
-the process is similar for other development setups.
-
-
-*Can I use a programming language other than Python to speed up my code?*
-
-Yes.  The languages used in SciPy are Python, Cython, C, C++ and Fortran.  All
-of these have their pros and cons.  If Python really doesn't offer enough
-performance, one of those languages can be used.  Important concerns when
-using compiled languages are maintainability and portability.  For
-maintainability, Cython is clearly preferred over C/C++/Fortran.  Cython and C
-are more portable than C++/Fortran.  A lot of the existing C and Fortran code
-in SciPy is older, battle-tested code that was only wrapped in (but not
-specifically written for) Python/SciPy.  Therefore the basic advice is: use
-Cython.  If there's specific reasons why C/C++/Fortran should be preferred,
-please discuss those reasons first.
-
-
-*How do I debug code written in C/C++/Fortran inside Scipy?*
-
-The easiest way to do this is to first write a Python script that
-invokes the C code whose execution you want to debug. For instance
-``mytest.py``::
-
-    from scipy.special import hyp2f1
-    print(hyp2f1(5.0, 1.0, -1.8, 0.95))
-
-Now, you can run::
-
-    gdb --args python runtests.py -g --python mytest.py
-
-If you didn't compile with debug symbols enabled before, remove the
-``build`` directory first. While in the debugger::
-
-    (gdb) break cephes_hyp2f1
-    (gdb) run
-
-The execution will now stop at the corresponding C function and you
-can step through it as usual. Instead of plain ``gdb`` you can of
-course use your favourite alternative debugger; run it on the
-``python`` binary with arguments ``runtests.py -g --python mytest.py``.
-
-
-*How do I enable additional tests in Scipy?*
```

Comment:
Information added to new [`doc\source\dev\contributor\runtests.rst`](https://github.com/scipy/scipy/pull/10286/files#diff-2dd64dc534d9aec2139341f236d14faf) linked from CTOC.